content/en/docs/ops/configuration/traffic-management/dns-proxy/test.sh

```diff
@@ -33,6 +33,13 @@ _verify_first_line snip_dns_capture_in_action_3 "$snip_dns_capture_in_action_3_o
 snip_address_auto_allocation_1
 _verify_contains snip_address_auto_allocation_2 "*   Trying 240.240."
 
+# verify external tcp services without vips
+snip_external_tcp_services_without_vips_1
+snip_external_tcp_services_without_vips_2
+snip_external_tcp_services_without_vips_3
+snip_external_tcp_services_without_vips_4
+_verify_like snip_external_tcp_services_without_vips_5 "$snip_external_tcp_services_without_vips_5_out"
```

Comment:
is this going to assume the IP output is stable? Might not be true

Reply:
@howardjohn "_verify_like" does not need exact IP... IP part can change. https://github.com/istio/istio.io/blob/master/tests/util/verify.sh#L428

Reply:
nice